src/match-iterator.coffee

```diff
@@ -166,7 +182,11 @@ class ForwardsMultiLine
       match
 
 class BackwardsMultiLine
-  constructor: (@buffer, @regex, range, @chunkSize) ->
+  constructor: (@buffer, @regex, range, @options={}, @chunkSize) ->
+    if _.isNumber(@options)
+      @chunkSize = @options
+      @options = {}
```

Comment:
I see what you're saying; thanks for being careful about compatibility. In this case though, these classes are private and their constructors don't have to have the same signature.